dist/templates/k8s.ovn.org_qoses.yaml.j2

```diff
@@ -0,0 +1,125 @@
+          is marked with relevant DSCP value and enforcing specified policing/shaping
+          parameters.
+        properties:
+          apiVersion:
+            description: |-
+              APIVersion defines the versioned schema of this representation of an object.
+              Servers should convert recognized schemas to the latest internal value, and
+              may reject unrecognized values.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources
+            type: string
+          kind:
+            description: |-
+              Kind is a string value representing the REST resource this object represents.
+              Servers may infer this from the endpoint the client submits requests to.
+              Cannot be updated.
+              In CamelCase.
+              More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: Spec defines the desired state of QoS
+            properties:
+              egress:
+                description: a collection of Egress QoS rule objects
+                items:
+                  properties:
+                    classifier:
+                      description: |-
+                        Classifer The classifier on which packets should match
+                        to apply the QoS Rule.
+                      properties:
+                        cidr:
+                          description: |-
+                            CIDR specifies the destination's CIDR. Only traffic heading
+                            to this CIDR will be marked with the DSCP value and policing/shaping
+                            get applies when specified.
+                            This field is optional, and in case it is not set the rule is applied
+                            to all egress traffic regardless of the destination.
+                          format: cidr
+                          type: string
+                        port:
+                          description: port that the traffic must match
+                          format: int32
+                          maximum: 65535
+                          minimum: 1
+                          type: integer
+                        protocol:
+                          description: protocol (tcp, udp, sctp) that the traffic
+                            must match.
+                          pattern: ^TCP|UDP|SCTP$
+                          type: string
+                      type: object
+                    markingDSCP:
+                      description: MarkingDSCP marking value for matching pods' traffic.
+                      maximum: 63
+                      minimum: 0
+                      type: integer
+                    policing:
```

Comment:
I think we should refrain from adding the policing/shaping stuff now. I just think in the future its going to be something we do and want to make sure the CRD will work for that as well.